node/api/client/renter.go

```diff
@@ -93,6 +94,18 @@ func (c *Client) RenterPricesGet() (rpg api.RenterPricesGET, err error) {
 	return
 }
 
+// RenterPostDownloadCacheSize uses the /renter endpoint to change the renter's
+// downloadCacheSize for streaming
+func (c *Client) RenterPostDownloadCacheSize(cacheSize uint64) (err error) {
```

Comment:
This should be `RenterSetStreamCacheSizePost`. I know that there some functions named `RenterPostXXX` but that was actually a mistake. It's not that easy in this case but usually a library method for an endpoint of the form `/end/point/path` should have the function name `EndPointPathPost` or `EndPointPathGet`.